backend/medtagger/api/scans/serializers.py

```diff
@@ -86,6 +87,7 @@
 in__label_model = api.model('Label model', {
     'elements': fields.List(fields.Raw, required=True),
     'labeling_time': fields.Float(description='Time in seconds that user spent on labeling', required=True),
+    'task_id': fields.Integer(description='ID of the Task that label refers to', required=True),
```

Comment:
For me it would be better to use key here. Input payload would be easier to read :)

**Proposal:** Maybe Task should be placed in the URL? This would be more RESTful.
`/<string:scan_id>/label` -> `/<string:scan_id>/<string:task_key>/label`